includes/class-demo-importer.php

```diff
@@ -1340,6 +1340,32 @@ public function refresh_demo_lists() {
 			wp_safe_redirect( admin_url( 'themes.php?page=demo-importer&browse=all' ) );
 		}
 	}
+
+    // Get page by title.
+	public function get_page_by_title( $title ) {
```

Comment:
Use php doc block comment instead of single line comment
https://docs.phpdoc.org/3.0/guide/references/phpdoc/basic-syntax.html

Reply:
Added docblock.